test/integration/admin_can_edit_any_item_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.